Gulpfile.js

```diff
@@ -9,30 +9,66 @@ ugifly = require('gulp-uglify');
 const libs = 'app/public/libs/'; 
 
 gulp.task('swigit', gulpSequence('lint', 'concat', ['minify-css', 'uglify']); 
+const gulp = require('gulp'),
+gulpSequence = require('gulp-sequence'),
+gulpIgnore = require('gulp-ignore'),
+jshint = require('gulp-jshint'),
+cleanCSS = require('gulp-clean-css')
+concat = require('gulp-concat'),
+uglify = require('gulp-uglify'); 
+
+const libs = './app/public/libs/'; 
+
+gulp.task('default', gulpSequence('concat', 
+  'lint', 
+  // 'minify-css'
+  ['minify-css', 'uglify']
+  )); 
+
+//For some reason, linting doesn't work if I pass in all our JS files.
+
+//I have it lint the concatenated file instead, which doesn't seem to hurt
+//the speed too much. 
 
 gulp.task('lint', function(){
-  return gulp.src(['./**/*.js']) //Takes in everything that's *not* in our public/lib folder 
+  return gulp.src(
+  'app/public/build/js') //Takes in everything that's *not* in our public/lib folder 
     .pipe(jshint())
-    .pipe(gulpIgnore.exclude(libs));
 }); 
 
+
 gulp.task('concat', function(){
-  return gulp.src(['./**/*.js'], '!./app/public/libs/angular', 
-  '!./app/public/libs/jquery', '!./app/public/libs/materialize') //Takes in everything that's *not* a library we're using
+  return gulp.src(['./**/*.js']) //Takes in all our JavaScript files 
   .pipe(concat('app.js'))
+  // .pipe(gulpIgnore).exclude('node_modules')
+  // .pipe(gulpIgnore).exclude('gulpfile.js')
+  .pipe(gulpIgnore.exclude('app/public/libs/angular')) //Excludes our CRM libraries 
+  .pipe(gulpIgnore.exclude('app/public/libs/jquery'))
+  .pipe(gulpIgnore.exclude('app/public/libs/materialize'))
   .pipe(gulp.dest('app/public/build/js')); 
-})
+}); 
+
+//NOTE: Concatenation *technically* seems to work, but takes about 6-7 seconds and produces a much 
+//larger file than we'd probably like (~60 MB in size!), which Sublime Text can't load (but TextEdit
+//can for some reason). The file doesn't appear to be ugified even though gulp claims that uglification
+//worked and doesn't seem to exclude some of the libraries we're trying to exclude (like jQuery). 
+
```

Comment:
I have a feeling the concat is pulling in the node_modules folder, and from what it looks like, a lot of the files in the libs folder as well. I'll clone your branch and play around with it tomorrow before we chat to see if I can find out what's going wrong. It would be nice to have this out of the way by the end of tomorrow so we can focus on wiring up the client to the backend.

Thanks for your work on this, shouldn't need much as far as fixes, and it'll be easy to build upon as we progress.